libs/SalesforceSDK/src/com/salesforce/androidsdk/config/BootConfig.java

```diff
@@ -98,6 +98,39 @@ public static BootConfig getBootConfig(Context ctx) {
 		}
 		return INSTANCE;
 	}
+
+	/**
+	 * Validates a boot config's inputs against basic sanity tests.
+	 * @param config The BootConfig instance to validate.
+	 * @throws BootConfigException If the boot config is invalid.
+	 */
+	public static void validateBootConfig(BootConfig config) {
```

Comment:
Should this be renamed to `validateHybridBootConfig`, since it's validating only hybrid config?

Reply:
I left it generic, because it could validate native-based configuration in the future.  I.e. if you call this for a native config at this point, it will still "validate"—there's just no validation.  But we are only calling it in the hybrid case at this point, so I could see it that way too.